submissions/ayodeleayobami/html/HTML TASK.html

```diff
@@ -0,0 +1,31 @@
+<!Doctype html>
```

Comment:
Please rename file to have no spaces. Refer to [Styleguide here](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19/blob/master/STYLEGUIDE.md
)

Reply:
"Doctype" should be all lowercase. Refer to Styleguide